br/pkg/lightning/backend/local/local.go

```diff
@@ -1145,14 +1145,16 @@ func (local *Backend) generateAndSendJob(
 	for _, jobRange := range jobRanges {
 		r := jobRange
 		eg.Go(func() error {
-			select {
-			case <-egCtx.Done():
+			if egCtx.Err() != nil {
 				return nil
-			default:
 			}
```

Comment:
any difference?

Reply:
`select {}` will lock the channel to read the value so performance is better. And code becomes shorter